src/main/java/edu/asu/epilepsy/apiv30/dao/impl/JdbcDAO.java

```diff
@@ -6,26 +6,11 @@
 import edu.asu.epilepsy.apiv30.dao.ValueObject;
 import edu.asu.epilepsy.apiv30.helper.APIConstants;
 import edu.asu.epilepsy.apiv30.helper.GsonFactory;
-import edu.asu.epilepsy.apiv30.model.ActivityInstance;
-import edu.asu.epilepsy.apiv30.model.ContainerActivity;
-import edu.asu.epilepsy.apiv30.model.MedicalAdherence;
+import edu.asu.epilepsy.apiv30.model.*;
```

Comment:
Java IDE like Intellij will love to use asterisks. However, this is a bad bad idea. Import all the resource from a lib will cause memory issue. Although the Java handle these import pretty well, but as a good developer, I suggest to remove the asterisk and replace with actual imports.

The solution is to go to Preferences (⌘ + , on macOS / Ctrl + Alt + S on Windows) > Editor > Code Style > Java > Imports tab set Class count to use import with '*' and Names count to use static import with '*' to a higher value. Any value over 99 seems to work fine.
source: https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports

Reply:
Thanks for that! I will make the required changes